com.microsoft.mrtk.input/Utilities/PoseSource/PolyfillHandRayPoseSource.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Subsystems;
+using System;
+using UnityEngine;
+using UnityEngine.XR;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    [Serializable]
+
```

Comment:
Extra line? Also I am not sure if we tend to put attributes directly above classes/fields @keveleigh

Reply:
Fixed